apps/worker/src/app/workflow/usecases/message-matcher/message-matcher.usecase.ts

```diff
@@ -406,7 +406,9 @@ export class MessageMatcher {
     if (fieldFilter.operator === 'IN') {
       result = actualValue.includes(filterValue);
     }
-
+    if (fieldFilter.operator === 'IS_DEFINED') {
+      result = actualValue !== undefined;
+    }
```

Comment:
Could we add a test case for this in one of the existing filters test on the API ?